### DISCUSSION TITLE
Add this repo to mpv.io

### DISCUSSION THREAD

**Discussion by Obegg** [zhongfly/mpv-build#13] [General]
Why not let more people know about this repo? It has auto-update and daily releases

**Comment by zhongfly** [comment]
1. I don't know how to add a link on mpv.io
2. I'm not a professional programmer, I don't know enough about compiling mpv and linux, I'm worried that I don't have enough ability to maintain this project for a long time.

**Reply by Obegg** [reply]
Uhm, I get it, mmmmm, then what about a collab with shinchiro? I already suggested it there, and it seems like shinchiro does know how to program, and he already has his builds on the official page of mpv, so it adds credibility and that he maintains the builds, and he has auto-update, the only "problem" is the release cycle, that's where you can help out. You managed to make it build mpv daily on GitHub, that's impressive (I am not a programmer), I'm sure that if you both work together it will result in fully autonomous builds, daily builds, and auto-update.

**Reply by zhongfly** [reply]
I'm more than willing to cooperate if shinchiro wants to.

**Reply by zhongfly** [reply]
I think you should close https://github.com/shinchiro/mpv-winbuild-cmake/issues/171 and open a new issue to request daily updates

**Reply by Obegg** [reply]
https://github.com/shinchiro/mpv-winbuild-cmake/issues/174#issuecomment-1025187292

**Reply by Obegg** [reply]
Can open a pull request as he suggested?

**Reply by zhongfly** [reply]
> Can open a pull request as he suggested?

I need to do some preparation

**Reply by Obegg** [reply]
Are the preparations done?

**Reply by zhongfly** [reply]
> Are the preparations done?

Due to being too busy and lacking time in real life, it will be postponed until I have enough time.

**Comment by hooke007** [comment]
Adding to mpv.io needs mpv devs' permission.

https://github.com/mpv-player/mpv.io/pull/92

**Reply by zhongfly** [reply]
Do you think I should add this repo on mpv.io?

**Reply by hooke007** [reply]
up to you

**Comment by avih** [comment]
> Do you think I should add this repo on mpv.io?

IMO not yet. It should demonstrate it's being maintained regularly for a good while, and gain enough popularity first, and then we'll see.

It's off to a good start though.

But also, since it's basically a cloned build setup of another project, it would probably be best if daily builds are merged into shinchiro's project.
